src/pages/workspace/members/WorkspaceOwnerPaymentCardForm.tsx

```diff
@@ -119,174 +75,79 @@ function WorkspaceOwnerPaymentCardForm({policy}: WorkspaceOwnerPaymentCardFormPr
-                        href={CONST.TERMS_URL}
-                    >
-                        {translate('workspace.changeOwner.addPaymentCardTerms')}
-                    </TextLink>{' '}
-                    {translate('workspace.changeOwner.addPaymentCardAnd')}{' '}
-                    <TextLink
-                        style={[styles.textMicroSupporting, styles.link]}
-                        href={CONST.PRIVACY_URL}
-                    >
-                        {translate('workspace.changeOwner.addPaymentCardPrivacy')}
-                    </TextLink>{' '}
-                    {translate('workspace.changeOwner.addPaymentCardReadAndAcceptTextPart2')}
-                </Text>
-                <Section
-                    icon={Illustrations.ShieldYellow}
-                    cardLayout={CARD_LAYOUT.ICON_ON_LEFT}
-                    title={translate('requestorStep.isMyDataSafe')}
-                    containerStyles={[styles.mh0, styles.mt5]}
-                >
-                    <View style={[styles.mt4, styles.ph2, styles.pb2]}>
-                        <Text style={[styles.searchInputStyle, styles.dFlex, styles.alignItemsCenter]}>
-                            <Icon
-                                src={Expensicons.Checkmark}
-                                additionalStyles={[styles.mr3]}
-                                fill={theme.iconSuccessFill}
-                            />
-                            {translate('workspace.changeOwner.addPaymentCardPciCompliant')}
-                        </Text>
-                        <Text style={[styles.mt3, styles.searchInputStyle, styles.dFlex, styles.alignItemsCenter]}>
-                            <Icon
-                                src={Expensicons.Checkmark}
-                                additionalStyles={[styles.mr3]}
-                                fill={theme.iconSuccessFill}
-                            />
-                            {translate('workspace.changeOwner.addPaymentCardBankLevelEncrypt')}
-                        </Text>
-                        <Text style={[styles.mt3, styles.searchInputStyle, styles.dFlex, styles.alignItemsCenter]}>
-                            <Icon
-                                src={Expensicons.Checkmark}
-                                additionalStyles={[styles.mr3]}
-                                fill={theme.iconSuccessFill}
-                            />
-                            {translate('workspace.changeOwner.addPaymentCardRedundant')}
-                        </Text>
-                    </View>
-                    <Text style={[styles.mt3, styles.searchInputStyle]}>
-                        {translate('workspace.changeOwner.addPaymentCardLearnMore')}{' '}
+        <PaymentCardForm
+            shouldShowPaymentCardForm={shouldShowPaymentCardForm}
+            addPaymentCard={addPaymentCard}
+            showCurrencyField
+            isDebitCard
```

Comment:
```suggestion
```
The form on this page should be the payment card, not debit card/wallet

Reply:
@amyevans are you sure?
Based on how was implemented WorkspaceOwnerPaymentCardForm - i see that it used debit card translation
<img width="1290" alt="Screenshot 2024-06-05 at 17 17 50" src="https://github.com/Expensify/App/assets/28352309/b5b71618-f570-4491-8e30-d976bb3d5717">

Reply:
Brought the question to [Slack](https://expensify.slack.com/archives/C036QM0SLJK/p1717606582669759)

Reply:
@amyevans updated